bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/aemform/AEMForm.java

```diff
@@ -144,6 +144,11 @@ default FormType getFormType() {
         throw new UnsupportedOperationException();
     }
 
+    @JsonIgnore
+    default String getRoleAttribute() {
+        throw new UnsupportedOperationException();
```

Comment:
All functions in interface should have proper java doc, see the other functions for reference java doc sample

Reply:
Added java doc